test/healthcheck/framework/utils.go

```diff
@@ -116,3 +117,26 @@ func endpointAvailable(c kubernetes.Interface, namespace, name string) wait.Cond
 		return true, nil
 	}
 }
+
+// logErrorf creates a new error using msg and param for the formated message.
+// The message is logged and the new error returned.  This function attempts to
+// log the location of the caller (file name & line number) so as to maintain
+// context of where the error occured
+func logErrorf(msg, param string) error {
+	_, file, line, _ := runtime.Caller(1)
+
+	// only use the last 30 characters
+	context := len(file) - 30
+	if context < 0 {
+		context = 0
+	}
+	partialFileName := file[context:]
+	format := fmt.Sprintf("...%s:%d: %v", partialFileName, line, msg)
+	e := fmt.Errorf(format, param)
+	glog.Error(e)
+	return e
+}
+
+func logError(msg string) error {
+	return logErrorf(msg, "")
```

Comment:
Won't `logErrorf()` now log this line instead of the line that calls `logError()`?

Reply:
gah, yep, thanks